bin/wgif

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env ruby
 
+$: << File.join(File.expand_path(__dir__), "..", "lib")
```

Comment:
I'm not even sure I knew `__dir__` was a thing. I think I tried to recreate this line from memory and lucked into `__dir__`. Also not sure how things were working for me with it going to the wrong place. Anyway, fixing now.